modules/bitcoind.nix

```diff
@@ -209,6 +217,34 @@ in {
         example = "tcp://127.0.0.1:28333";
         description = "ZMQ address for zmqpubrawtx notifications";
       };
+      assumevalid = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        example = "00000000000000000000e5abc3a74fe27dc0ead9c70ea1deb456f11c15fd7bc6";
+        description = "If this block is in the chain assume that it and its ancestors are valid and potentially skip their script verification";
+      };
+      addnodes = mkOption {
+        type = types.listOf types.str;
+        default = [];
+        example = [ "ecoc5q34tmbq54wl.onion" ];
+        description = ''
+          Add nodes to connect to and attempt to keep the connections open. Will
+          add one addnode= entry to the config per array element.
```

Comment:
Nit: `array` is `list` in nix lingo.
I think the implementation specifics here are obvious and don't warrant further explanation. So I'd remove the whole `Will add...` sentence.